public/Sjofn/homework1.html

```diff
@@ -0,0 +1,72 @@
+<html lang="en">
+<head>
+    <title>Homework 1</title>
+    <style>
+        div {
+            background-color: dodgerblue;
+        }
+        #myId{
+            color: tomato;
+            background-color: cadetblue;
+            size: A3;
+        }
+        .myClass{
+            width: 300px;
+        }
+    </style>
+</head>
+<body>
+<div>
+    <i>
+        <ol>
+            <li>Implement a HTML page. Choose one of the following topics
+                <ul>
+                    <li>a.TOP 5 programming languages I would like to learn</li>
+                    <li>b.TOP 5 countries I would like to visit</li>
+                </ul>
+            </li>
+            <li>Requirements:
+                <ul>
+                    <li>- ‘Homework 1’ is title of the page</li>
+                    <li>- A page contains different ‘div’ sections. Each one starts with corresponding heading and ends with ‘hr’</li>
+                    <li>- Section 1. Copy this homework’s task and represent it as italic text. Preserve structure (lists) and line breaks</li>
+                    <li>- Section 2. Enumerate your items in a list. Name of each item should be a link to an article in the internet. A link should open in a new tab</li>
+                    <li>- Section 3. Pick one of items and provide some information for it, in 2-3 sentences in a paragraph. Highlight 3 important words/phrases with special color, ‘underline’ and ‘bold’ styles, accordingly. Put the paragraph in a new ‘div’ with maximum width 300px (set this style via CSS)</li>
```

Comment:
![pic1](https://user-images.githubusercontent.com/16742190/73011755-8e43c080-3e1d-11ea-9cd9-7aa6b9adcdb6.JPG)
Please, check the screenshot. Many symbols are displayed incorrectly (Chrome browser). 
Looks like a good opportunity to use `<meta>` tag with encoding provided:
https://www.w3schools.com/tags/att_meta_charset.asp